values-zh-rCN/strings.xml

```diff
@@ -92,42 +92,42 @@
             <string name="label_decline">拒绝</string>
             <string name="label_delete">删除</string>
-            <string name="label_delete_warning">Delete %1$s?</string>
-            <string name="dont_show_again">Don’t show again</string>
-            <string name="title_optional_app_update">发现 App 更新</string>
+            <string name="label_delete_warning">确定删除 %1$s 吗？</string>
+            <string name="dont_show_again">不再提示</string>
+            <string name="title_optional_app_update">发现新版本 App</string>
             <string name="ms_prev">上一步</string>
             <string name="ms_next">下一步</string>
             <string name="ms_end">完成</string>
         <!--FEEDBACK-->
-            <string name="label_now">Now</string>
-            <string name="label_later">Later</string>
-            <string name="label_never">Never</string>
-            <string name="label_rate">Rate</string>
-            <string name="title_rate_us">Rate this app</string>
-            <string name="message_rate_us">We hope you enjoy using this app, it won’t take more than a minute to rate or give some feedback. It can help other users discover this app too!</string>
-            <string name="label_supportproject">Support</string> <!--verb-->
-            <string name="message_supportproject">If you like the Open GApps packages, please consider supporting the project. The development of the packages is completely open source and we all benefit from the developers’ efforts!</string>
+            <string name="label_now">立刻</string>
+            <string name="label_later">以后</string>
+            <string name="label_never">从不</string>
+            <string name="label_rate">评价</string>
+            <string name="title_rate_us">评价这个 App</string>
+            <string name="message_rate_us">我们希望你能喜欢这个 App。不妨花一分钟时间评价一下或者提供一些反馈意见，从而让更多的人发现这个 App！</string>
+            <string name="label_supportproject">支持</string> <!--verb-->
+            <string name="message_supportproject">如果你喜欢 Open GApps 安装包，请考虑支持该项目。这些安装包的开发完全开源，我们所有人都会从众多开发者的努力中受益！</string>
         <!--EXPLANATION-->
             <string name="explanation_permission_denied">为了 App 正常工作，你必须授权 Open GApps 读写你的手机存储。言之有理，不是吗？</string>
             <string name="explanation_adblock_detected">检测到 Adblocker!? 广告令人不爽，但是它们为项目和开发者提供了经济支持。请考虑禁用 Adblocker 或者捐助，谢谢！</string>
-            <string name="title_webview_not_installed">Oops! No WebView Provider installed!?</string>
-            <string name="message_webview_not_installed">It seems you have uninstalled \"Stock WebView\", but your ROM does not support the \"Google WebView\". Re-flash your ROM and ask the ROM developer to add support for alternative WebView Providers.</string>
+            <string name="title_webview_not_installed">天呐！没有安装 WebView 组件!?</string>
+            <string name="message_webview_not_installed">你似乎卸载了手机自带的 WebView，并且你的 ROM 不支持“Android 系统 WebView”。你可以尝试重刷 ROM 或者寻求 ROM 开发者加入 WebView 组件支持。</string>
```

Comment:
Would it be possible to still (also) use the terms for "Stock WebView" for the one that comes with the rom, and the "Google WebView" from the Open GApps package? That could give the user more information about the keywords and understanding the debug log.